### ISSUE TITLE
Custom block <docs></docs> not working when developing

### ISSUE THREAD

**Issue by rjoo** [vue-styleguidist/vue-cli-plugin-styleguidist#15]
In a fresh Vue CLI 3 project, I ran `vue add styleguidist`, it installs everything correctly and styleguidist runs great. But during development with `npm run serve`, I get a `Module parse failed: Unexpected token` compile error for anything within the custom `<docs>` block in a Vue component file.

Before digging into webpack and vue-loader and setting a loader around `<docs>`, I was hoping this would be more out of the box (since it installs with an example AppButon.vue with `<docs>`) and I'm just missing a simple setting.

**Comment by elevatebart** [comment]
As a temporary workaround you can use the following loader in your vue.config.js
```js
 configureWebpack: {
   module: {
      rules: [
        {
          resourceQuery: /blockType=docs/,
          loader: "null-loader"
        }
      ]
    }
  }
```
I must add that you need to add a null loader to your dependencies

**Comment by elevatebart** [comment]
fixed in 0.1.6 ;)

**Comment by elevatebart** [comment]
I believe this is fixed. Closing.
